libs/plugins/scully-plugin-md-replace-route/README.md

```diff
@@ -0,0 +1,7 @@
+# plugins-scully-plugin-md-replace-route
+
+This library was generated with [Nx](https://nx.dev).
```

Comment:
remove the default content, and add some simple docs or a link to the place the docs end up

Reply:
I'm still missing the plugin's description here